provisioning/virtual-machines/ansible/kubernetes/tasks/main.yaml

```diff
@@ -0,0 +1,96 @@
+---
+# Install kubernetes (not configure the cluster)
+
+  
+- name: Create docker daemon directory
+  file:
+    path: /etc/docker
+    state: directory
+
+- name: Create docker daemon file
+  copy:
+    src: files/daemon.json
+    dest: /etc/docker/daemon.json
+    owner: root
+    group: root
+    mode: '0644'
+  notify: Restart the docker service
+
+- name: Install resolvconf
+  apt:
+    name: "{{resolvconf}}"
+    state: latest
+    force_apt_get: yes
+  vars:
+    resolvconf:
+    - resolvconf
+
+- name: Resolvconf directory
+  file:
+    path: /etc/resolvconf/resolv.conf.d
+    state: directory
+
+- name: Set DNS to 8.8.8.8
+  copy:
+    content: "nameserver 8.8.8.8\n"
```

Comment:
I don't think that 8.8.8.8 this flow is whitelisted in LADISPE

Reply:
Use 130.192.3.21 (our main DNS server)?

Reply:
I would just delete this part and test everything with the default settings (i.e. using coredns as the main DNS server, which in turn forwards the requests to the Polito DNS servers).